artemis/src/test/resources/complete_config.toml

```diff
@@ -1,110 +1,46 @@
-## CURRENT VALID TOML FILE
-
-[node]
```

Comment:
Most renamed options contain the hierarchy label in them... i.e. `port` is now `p2p-port`, `metrics-port`, `rest-api-port` i.e. you don't need a hierarchy. Either way, I have no strong opinion here.

I think the plan is to move to yaml eventually.

@ajsutton any opinion?

Reply:
Seems like we could keep the hierarchical structure, so that in the toml file, you get the nesting but at the cli level the nesting is flattened so that `p2p-port` would be written to the toml file as:
```
[p2p]
port = 9000
```

But if we're moving away from toml anyway, maybe it doesn't matter.  🤷‍♀ Just want to make sure we're being intentional with how this is structured.